lib/routes/devices-and-sessions.js

```diff
@@ -245,11 +245,13 @@ module.exports = (log, db, config, customs, push, devices) => {
           payload: isA.alternatives().try(
             isA.object({
               to: isA.string().valid('all').required(),
-              _endpointAction: isA.string().valid('accountVerify').optional(),
+              _endpointAction: isA.string().valid('accountVerify', 'sendtab').optional(),
               excluded: isA.array().items(isA.string().length(32).regex(HEX_STRING)).optional(),
-              payload: isA.object().when('_endpointAction', { is: 'accountVerify', then: isA.required(), otherwise: isA.required() }),
+              payload: isA.object().when('_endpointAction', { is: 'accountVerify', then: isA.required() }),
+              encryptedPayload: isA.string().regex(HEX_STRING).when('_endpointAction', { is: 'sendtab', then: isA.required(), otherwise: isA.forbidden() }), // TODO: THIS SHOULD BE BASE64! See https://docs.google.com/document/d/1YT6gh125Tu03eM42Vb_LKjvgxc4qrGGZsty1_ajf2YM/edit?disco=AAAABrAK_y0
+              pushboxToken: isA.string().regex(HEX_STRING).when('encryptedPayload', { is: isA.exist(), then: isA.required(), otherwise: isA.forbidden() }),
```

Comment:
This is good for getting up and running, but since this endpoint is authenticated with a `sessionToken`, we should be able to mint an OAuth token on the backend rather than requiring the client to pass one in manually.  Might have to wait until we solve https://github.com/mozilla/fxa-auth-server/issues/2317 though.